### DISCUSSION TITLE
how to use torch compile with lora?

### DISCUSSION THREAD

**Discussion by frankyifei** [comfyanonymous/ComfyUI#5472] [General]
I could not get lora to work with torch compile, only checkpoints work,

**Comment by LukeG89** [comment]
At the moment you can't, because as Kijai said in this [Reddit post:](https://www.reddit.com/r/comfyui/comments/1gdeypo/i_cant_get_torch_compile_to_work_with_lora_it/)

> LoRAs do not currently work with torch.compile because the torch.compile happens before LoRA weights are loaded (in the core code, nothing to do with the node order in the graph).

> I modified that bit of the code to swap the order of things, and it seems to work then. This might break some other stuff, so I've let ComfyUI team know and we'll probably get better way to deal with this issue soon.

**Comment by LukeG89** [comment]
@frankyifei  I don't use torch compile, but I guess [ComfyUI-KJNodes](https://github.com/kijai/ComfyUI-KJNodes) just found a solution for that: https://github.com/kijai/ComfyUI-KJNodes/commit/8cfbbaf29e0ee44ab8ec0155d65e6f510549c599

Try `Patch Model Patcher Order` node from KJNodes.
